### PR TITLE
Migrate docs from Wiki to GitHub

### PR DESCRIPTION
**Pull request by sladyn98**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+## Changelog
+
+### Version 1.6 and later
+
+See [GitHub
+releases](https://github.com/jenkinsci/antisamy-markup-formatter-plugin/releases){.external-link}.
```
Comment (oleg-nenashev) [inline CHANGELOG.md]:
```suggestion
releases](https://github.com/jenkinsci/antisamy-markup-formatter-plugin/releases).
```

File: README.md [hunk 2]
```diff
@@ -22,6 +22,29 @@ which uses the [`MyspacePolicy`](src/main/java/hudson/markup/MyspacePolicy.java)
...
+When installed, 'Safe HTML' can be selected as Markup Formatter in
+"Manage Jenkins" → "Configure Global Security" → "Markup Formatter":
+
+![](docs/images/safeHtml.png){.confluence-embedded-image
+.confluence-content-image-border}
+
+User-submitted text will be sanitized by removing potentially dangerous
+elements.
+
+![](docs/images/sanitized.png){.confluence-embedded-image
```
Comment (oleg-nenashev) [inline README.md]: Confluence macros can be deleted

File: README.md [hunk 3]
```diff
@@ -22,6 +22,29 @@ which uses the [`MyspacePolicy`](src/main/java/hudson/markup/MyspacePolicy.java)
...
 
+## Configuration
+
+This plugin is bundled in the Jenkins WAR file and will generally be
+preinstalled.
+
+When installed, 'Safe HTML' can be selected as Markup Formatter in
+"Manage Jenkins" → "Configure Global Security" → "Markup Formatter":
+
+![](docs/images/safeHtml.png){.confluence-embedded-image
```
Comment (oleg-nenashev) [inline README.md]: Confluence macros can be deleted

**Comment by oleg-nenashev** [comment]
Thanks @sladyn98 !

File: README.md [hunk 4]
```diff
@@ -2,25 +2,27 @@
...
+
+When installed, 'Safe HTML' can be selected as Markup Formatter in "Manage Jenkins" → "Configure Global Security" → "Markup Formatter":
+
+User-submitted text will be sanitized by removing potentially dangerous elements.
+
+## Changing or altering the policy
+
+At least in 1.5, the "Safe HTML" plugin has no support for editing, overriding, or updating the HTML sanitization policy. A custom plugin must be built instead. See `hudson.markup.RawHtmlMarkupFormatter.java` .
 
 # See also:
```
Comment (jglick) [inline README.md]: Delete the wiki link, and probably also delete the plugin site link (or replace with a badge).

**Pull request merged**